webapp/channels/src/components/admin_console/billing/billing_history.tsx

```diff
@@ -39,6 +39,7 @@ export const NoBillingHistorySection = (props: NoBillingHistorySectionProps) =>
             />
         </div>
         <ExternalLink
+            data-testid='BillingHistory-link'
```

Comment:
I know we're not consistent but our convention for `data-testid` and element ID are `camelCase`. Would be great if this and others here are updated.